digipost-vilkaar-bedrift.md

```diff
@@ -130,7 +130,7 @@ Kunden samtykker til at Posten benytter underleverandører for utførelsen av si
 
 Som ledd i Postens oppfyllelse av avtalen med Kunden om bruk av Digipost vil Posten behandle personopplysninger relatert til Kunde, herunder om ansatte eller brukere Kundens konto. 
 
-Behandlingsansvarlig er i slike tilfeller Posten Norge AS, Att: Digipost, Postboks 1500 Sentrum, 0001 Oslo, +47 23 14 90 00. 
+Behandlingsansvarlig er Posten Bring AS, Postboks 1500 Sentrum, 0001 Oslo.
```

Comment:
Denne teksten samsvarte ikke med tekst i Post-It og har blitt oppdatert tilsvarende. Si ifra om det heller skal oppdateres andre veien så vi beholder som det var, og Post-It da får tilleggsteksten.

Reply:
her tenker jeg at Hege, eller noen andre, kan ha noen meninger

Reply:
Jeg støtter endringen du har lagt inn her. Vi trenger ikke tilleggsteksten.